libs/vkd3d/command.c

```diff
@@ -4929,16 +4929,65 @@ static bool vk_image_memory_barrier_from_d3d12_transition(const struct d3d12_dev
     return vk_barrier->oldLayout != vk_barrier->newLayout;
 }
 
+static void vk_image_memory_barrier_for_after_aliasing_barrier(struct d3d12_device *device,
+        struct d3d12_resource *after, VkImageMemoryBarrier *vk_barrier)
+{
+    const struct vkd3d_format *vk_format = vkd3d_get_format(device, after->desc.Format, false);
```

Comment:
The depth/stencil argument is a bit confusing, I think this is the right way to do this?

Reply:
Honestly I'm not sure, ideally we'd store the vkd3d_format pointer with the resource....

Reply:
Looks correct, the depth stencil variant is there to have different typeless overrides.